### DISCUSSION TITLE
Cant start quarkus on port 80 unless I run with sudo

### DISCUSSION THREAD

**Discussion by newk5** [quarkusio/quarkus#29936] [Q&A] [answered]
I get the error:

```
Port 80 seems to be in use by another process. Quarkus may already be running or the port is used by another application.
```
When I try to run quarkus normally: java -jar quarkus-run.jar
But there is nothing running on port 80, I've double and triple checked this. 
If I run the same command with sudo, the application starts normally. I've never had this issue before and usually run all my quarkus apps without sudo just fine, so what could be causing this to happen?

This is on Ubuntu 22, Quarkus 2.6.2. and the VM is behind a very restrictive firewall but can be accessed from outside on port 80 as long I run the app with sudo. Also if I change the port from 80 to 8080 for example, I'm able to start it without sudo so its very strange.

**Comment by maxandersen** [comment, marked as answer]
That is normal Linux behavior not tied to Quarkus. Running any process on ports below 1024 normally requires root access. 

This article explains and provide some solutions that might work for you: https://serverfault.com/questions/112795/how-to-run-a-server-on-port-80-as-a-normal-user-on-linux

**Reply by newk5** [reply]
Oh wow TIL. Apologies, was not aware of that, thank you for the link, will look into it.
